dags/imagegen_devx/configs/project_bite_config.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities to construct configs for solutionsteam_jax_bite DAG."""
+
+
+from typing import Tuple
+from xlml.apis import gcp_config, metric_config, task, test_config
+from dags import gcs_bucket, test_owner
+from dags.imagegen_devx.configs import common
+from dags.vm_resource import TpuVersion, Project
+
+
+GCS_SUBFOLDER_PREFIX = test_owner.Team.IMAGEGEN_DEVX.value
+
+
+def set_up_axlearn() -> Tuple[str]:
+  return (
+      common.UPGRADE_PIP,
+      *common.set_up_nightly_jax(),
+      "python -m pip install axlearn",
+      "git clone https://github.com/apple/axlearn.git",
```

Comment:
Depending on whether we want to test against HEAD, we can also consider cloning followed by pip install. 

We also can consider using `pip install .[tpu]` to install axlearn + compatible TPU deps, since we may not always be developing against jax nightly.

Reply:
Thanks @markblee! My understanding for tests are nightly against HEAD. 

Could you help clarify the test requirements? i.e. HEAD or release version; nightly or released JAX version.

Reply:
@markblee I think our intention is to test JAX nightly against AXLearn HEAD, in order to catch errors before JAX releases. Let me know if you want to do something different!